scripts/build-controller-release.sh

```diff
@@ -187,60 +186,60 @@ if [ -z "$ACK_METADATA_CONFIG_PATH" ]; then
 fi
 
 helm_output_dir="$SERVICE_CONTROLLER_SOURCE_PATH/helm"
-ag_args="$SERVICE $RELEASE_VERSION -o $SERVICE_CONTROLLER_SOURCE_PATH --template-dirs $TEMPLATES_DIR --aws-sdk-go-version $AWS_SDK_GO_VERSION"
```

Comment:
Funny, my `shellcheck` didn't make this suggestion (replacing this string with an array).  Curious what command line you used?

Reply:
It was actually L211 where it complained about this. Because we are trying to expand a string to use as CLI args, it suggested we use an array of strings rather.